README.md

```diff
@@ -102,6 +102,17 @@ This gRPC server is started by the driver running in _Node_ mode and implements
 The following diagram illustrates the communication channels between driver components:
 ![communication diagram](/docs/images/communication/pmem-csi-communication-diagram.png)
 
+### Security
+
+All communication is protected by mutual TLS. Both client and server must identify themselves and the certificate they present must be trusted. The common name in each certificate is used to identify the different components. The following common names have a special meaning:
+
+- `pmem-registry` is used by the [RegistryServer](#node-registry-server).
+- `pmem-node-controller` is used by [NodeControllerServers](#node-controller-server)
+
```

Comment:
Can you add a comment explaining who accepts connections from whom? "All communication" is a bit vague. Once this is spelled out explicitly, it becomes possible to check that the code follows that specification and has the necessary tests.

Reply:
I feel other sections(Components and Communication channels) explains the server and clients involved in the driver. I update this section with references to them, hope this will improves bit.

Reply:
So the complete list of TCP connections is this?

- pmem-csi-driver -> pmem-registry
- pmem-csi-driver (controller) -> pmem-csi-driver (node)

Testing covers only one of these two connections. Handling of connections to the pmem-csi-driver still needs to be tested.

Reply:
yes, you are right, tests did not covered yet Controller -> Node connection,